### ISSUE TITLE
Flashing Image still not working in update v8.2.1

### ISSUE THREAD

**Issue by BlackGrimReaper** [renpy/renpy#5520]
It's a copy of issue #4994 and it's marked as completed. I tried using the code again in versions 8.2.1.24030407 and 8.3.0.24051701+nightly, but it's not working. Yes, it works flawlessly in 8.1.1. I assumed since it's marked as completed, it was fixed. Am I missing something? I recreated this issue because the previous one is marked as completed. I'm not sure if my response in that one will get any attention from the developers. 
Thank you.

**Comment by danac78** [comment]
I basically tried to repeat what you did. I defined the flash like so:

```
image testflash:
    "#00000000"
    block:
        alpha 0.75
        "#fff" with Dissolve(0.4) 
        0.4
        "#00000000" with Dissolve(0.4)
        0.5
        repeat
```

Wrote the code like this:

```
label testsystem:
    scene n2 constance dm graveyard 01
    with dissolve
    show lily mystique yeaaboutthat
    with dissolve
    pause
    show testflash
    pause
    hide testflash
    pause
return
```

https://www.dropbox.com/scl/fi/aowreailexf9and9alswg/testflash.mp4?rlkey=3cau2jttthd73anytuk84fhah&dl=0

Got this as a result. Based on what you said in both issues, this is the expected result. So if it is not the expected result, you might need to make it a little clearer. :)

**Comment by BlackGrimReaper** [comment]
> I think one thing that is missing is what "It is not working" means. Because I ran the code on 8.2 (with image transblack = "#00000000" instead of transparentimage.webp) because I am not going to make a webp for that. ;) The results when I use show is that it shows a flashing white over the the previous image (I use scene a lot since I don't use sprites).

The code works when I replace "transparentImage.webp" with "#00000000," but it fails when using an actual transparent image. While my issue is resolved using "#00000000" (thanks a lot, @danac78), the original code with the transparent image still doesn't function properly. Should I close this issue as resolved or not? Once again, thank you so much, @danac78 for the quick and effective solution.

**Comment by danac78** [comment]
I think using #00000000 or (image transcolor = "#00000000" would be better than having a file because it uses the color displayable, which does not require a image file. ;) I tried it with a webp, and it works fine. So, I just have a question where the webp file is and maybe changing it to a lowercase so you can do "transparentimage". The way you have it in the ATL is that it will try to find it in the root images folder (it will not look at subfolders of images for TransparentImage.webp). 

However, if you use the color code for it, it will work the same without needing a dedicated image file.:)

**Comment by danac78** [comment]
For testing clarification, I made a fresh webp from Photoshop with just a transparent background. I believe Photoshop native webp uses the standard encoding.

**Comment by BlackGrimReaper** [comment]
After some testing, I found the issue. My VN resolution is 1080p, but the transparent image I was using was 2160p, which caused the code to fail. When I switched to a 1080p image, it worked perfectly. The code did function with the 2160p image in version 8.1.1. Interestingly, in version 8.3, the code works with other 2160p images but not with the transparent one. It's a bit strange, but at least now I understand the problem.

**Comment by danac78** [comment]
Does it change if you take the 2160 webp and rename it TransparentImage@2.webp? Basically, renpy would see @2 and supersample it. So if it would show as if it was a 1080 as a displayable.

**Comment by BlackGrimReaper** [comment]
> Does it change if you take the 2160 webp and rename it [TransparentImage@2.webp](mailto:TransparentImage@2.webp)? Basically, renpy would see @2 and supersample it. So if it would show as if it was a 1080 as a displayable.

Yes, the code works when a 2160p webp file is renamed with the @2 suffix, but it doesn’t work without it.

**Comment by BlackGrimReaper** [comment]
> You may want to try a bunch of these: https://www.renpy.org/doc/html/incompatible.html Particularly, this one seems relevant: define config.transitions_use_child_placement = False

Nope, "define config.transitions_use_child_placement = False" doesn't work either.

**Comment by danac78** [comment]
Does this affect sprites at 2160 (because they would have transparency)? I am trying to determine if there is a problem with transparency at a higher resolution than the VN...or just when the whole file is a transparent image at a higher resolution than the VN.

**Comment by BlackGrimReaper** [comment]
> Does this affect sprites at 2160 (because they would have transparency)? I am trying to determine if there is a problem with transparency at a higher resolution than the VN...or just when the whole file is a transparent image at a higher resolution than the VN.

I tried the code with a 2160p partially transparent image, and the behavior is a bit weird. The code should display a white flash followed by the partially transparent image after 0.75 seconds, but instead, only the partially transparent image is blinking. In simpler terms, the partially transparent image appears and disappears after 0.75 seconds, but there's no white flash.

**Comment by danac78** [comment]
On the one hand, there is a conflict between the vn and image resolutions. On the other hand, it can be solved using the web color code shown above (it will be drawn at vn resolution) or supersampling (@2 or @4 or @6). The good thing about supersample is that it will appear as a VN resolution but have higher resolution details.

**Comment by renpytom** [comment]
I don't think this is actually an issue with Ren'Py.
